spec/lib/gh/pr.py

```diff
@@ -133,6 +133,20 @@ def is_approved(self) -> bool:
         ))
         return status == "APPROVED"
 
+    def is_passing_checks(self) -> bool:
+        status = sh.stdout((
+            "gh",
+            "pr",
+            "checks",
+            self.url,
+            "--json",
+            "state",
+            "--required",
+            "--jq",
+            'all(.[]; .state == "SUCCESS" or .state == "SKIPPED")',
+        ))
+        return status == "true"
```

Comment:
IS this used only in tests ? I cannot see it anywhere else

Reply:
yes - this whole file is only a helper for tests